cli/status/status.go

```diff
@@ -20,35 +68,110 @@ type StatusOpts struct {
 func Status(runningCtx running.RunningCtx, opts StatusOpts) error {
 	ts := table.NewWriter()
 	ts.SetOutputMirror(os.Stdout)
-	ts.AppendHeader(table.Row{"INSTANCE", "STATUS", "PID", "MODE"})
+	ts.AppendHeader(table.Row{"INSTANCE", "STATUS", "PID", "MODE", "ALERTS"})
 
 	for _, run := range runningCtx.Instances {
 		row := []interface{}{}
 		fullInstanceName := running.GetAppInstanceName(run)
 		procStatus := running.Status(&run)
-
 		row = append(row, fullInstanceName)
-		row = append(row, procStatus.ColorSprint(procStatus.Status))
-		if procStatus.Code == process_utils.ProcessRunningCode {
-			row = append(row, procStatus.PID)
-		}
+		warnings := []string{}
+		mode_str := ""
 
 		conn, err := connector.Connect(connector.ConnectOpts{
 			Network: "unix",
 			Address: run.ConsoleSocket,
 		})
 		if err == nil {
-			res, err := conn.Eval("return (type(box.cfg) == 'function') or box.info.ro",
+			res, err := conn.Eval("return require('config'):info()",
```

Comment:
We're doing 4 network requests plus one for each upstream. It means at least 3 round-trip-time waiting for each of the instances.

We can coalesce several requests into one:

```lua
eval("return <..expr X..>")
eval("return <..expr X..>")
eval("return <..expr X..>")

=> 

eval("do
    local x = <..expr X..>
    local y = <..expr Y..>
    local z = <..expr Z..>
    return x, y, z
end")
```

In fact, all the expressions are taking data from the `box.info` result. I guess we can call it once and parse on the client then.

I see that the RO/RW mode request has `type(box.cfg) == 'function'` check. I guess it is an attempt to handle a situation, when `box.cfg()` is not called yet. If so, we should keep this check. It should be verified on tarantool that has no `box.info` before first `box.cfg`. See https://github.com/tarantool/tarantool/issues/7255.

Reply:
Replaced with one `Eval` call.